inode2prog.cpp

```diff
@@ -225,7 +225,9 @@ struct prg_node * findPID (unsigned long inode)
 		return node;
 	}
 
-	reread_mapping();
+	#ifndef __APPLE__
+		reread_mapping();
```

Comment:
If I understand correctly, this means on apple you only see connections, not processes. Is that accurate?

I suppose that's better than nothing, but of course nethogs' selling point is that you see the processes :)